R/theme-defaults.r

```diff
@@ -98,9 +98,9 @@ theme_grey <- function(base_size = 11, base_family = "") {
     legend.key.size =    unit(1.2, "lines"),
     legend.key.height =  NULL,
     legend.key.width =   NULL,
-    legend.text =        element_text(size = rel(0.8)),
+    legend.text =        element_text(size = rel(16/11)),
     legend.text.align =  NULL,
-    legend.title =       element_text(hjust = 0),
+    legend.title =       element_text(size = rel(16/11), hjust = 0),
```

Comment:
Hi @tdhock thanks for the info! I did not think of this situation before.
For here, I believe it's because the plot is printed by the data generated by `ggplot_build` and did not render to html, so the text size gets bigger in ggplot. 

I changed the default text sizes for the convenience of rendering html, since previously animint set legend text size to 16px and I just thought the change to `rel(16/11)` would pass 16 as default to html. 

Yes, I think we should change the defaults back in ggplot default, and change our default text size in JS side.

Reply:
the top image was animint/html and the bottom was static/ggplot_build/png.
that is a good idea to change back the defaults in R code (so then static/ggplot_build/png looks the same), and also to change the defaults in JS (so the animint/html is more reasonably sized). Please create another PR for that.